UnityProject/Assets/Scripts/Objects/Medical/Genetics/GeneticEggLogic.cs

```diff
@@ -11,9 +11,7 @@ public class GeneticEggLogic : MonoBehaviour
 
 	public int ChosenHatchTime;
 
-	public List<GameObject> Mobs000 = new List<GameObject>();
-	public List<GameObject> Mobs100 = new List<GameObject>();
-	public List<GameObject> Mobs200 = new List<GameObject>();
+	public List<GameObject> ALL = new List<GameObject>();
```

Comment:
the hell is ``ALL``

Reply:
all mobs

Reply:
maybe remember it to be more clear then?